Uploader.py

```diff
@@ -0,0 +1,120 @@
+from wikidataStuff.WikidataStuff import WikidataStuff as WDS
+import pywikibot
+import importer_utils as utils
+from os import path
+
+MAPPING_DIR = "data"
+PROPS = utils.load_json(path.join(MAPPING_DIR, "properties.json"))
+
+SUMMARY = "nature test"
```

Comment:
I ended up inserting two variants of square tags, a default one for sandbox and a placeholder for the real one, when it's been decided upon.